src/components/Playback/player.css

```diff
@@ -1,10 +1,13 @@
 .player-wrap__main_controls .playkit-bottom-bar,
 .player-wrap__main_controls .playkit-pre-playback-play-overlay {
-    left: 50% !important;
+    width: 200% !important;
```

Comment:
would be great if we can avoid all of the `!important` in order to avoid future conflicts.
also, it's a bad practice.

Reply:
I know.. no choice here - I need to style player elements